packages/rum/src/performanceCollection.ts

```diff
@@ -7,6 +7,52 @@ interface BrowserWindow extends Window {
   PerformanceObserver?: PerformanceObserver
 }
 
+export interface RumPerformanceResourceTiming {
```

Comment:
What about extending existing interfaces?

Reply:
Having simpler interfaces describing what is actually used makes those data structure creation easier and type checked (no need to cast as much as before), for example the initial document and navigation timings creation are actually type-checked: https://github.com/DataDog/browser-sdk/blob/87c68d74f0c5be293a20b50bf9b6bb15ce2c7d73/packages/rum/src/performanceCollection.ts#L101-L143

Reply:
OK, let's always cast early native type to RUM type to work with the same types.
cf resourceUtil `entry: RumPerformanceResourceTiming | PerformanceResourceTiming`